lms/djangoapps/teams/static/teams/js/collections/base.js

```diff
@@ -1,18 +1,39 @@
 ;(function (define) {
     'use strict';
-    define(['common/js/components/collections/paging_collection'],
+    define(['paging-collection'],
         function(PagingCollection) {
             var BaseCollection = PagingCollection.extend({
-                initialize: function(options) {
-                    this.url = options.url;
+                queryParams: {
+                    totalPages: null,
+                    totalRecords: null
+                },
 
-                    PagingCollection.prototype.initialize.call(this);
+                constructor: function (models, options) {
+                    this.options = options;
+                    this.url = options.url;
+                    this.state.perPage = options.per_page;
 
                     this.course_id = options.course_id;
-                    this.perPage = options.per_page;
-
                     this.teamEvents = options.teamEvents;
                     this.teamEvents.bind('teams:update', this.onUpdate, this);
+
+                    this.queryParams = _.extend({}, BaseCollection.prototype.queryParams, this.queryParams);
+                    PagingCollection.prototype.constructor.call(this, models, options);
+                },
+
+                // Review Note: Had to overwrite this because if empty response is returned from server
+                // PagingCollection fails to parse it. PagingCollection expects the
+                // results key in object along with other meta keys
+                parse: function (response, options) {
+                    if (!response) {
+                        response = {};
+                    }
+
+                    if (!response.results) {
+                        response.results = [];
+                    }
+
+                    return PagingCollection.prototype.parse.call(this, response, options);
                 },
```

Comment:
Had to overwrite this because if empty response is returned from server PagingCollection fails to parse it. PagingCollection expects the results key in object along with other meta keys